simtools/data_model/validate_schema.py

```diff
@@ -163,8 +163,8 @@ def _validate_data_type(self, schema, key, data_field):
             self._validate_datetime(data_field, self._field_is_optional(schema))
         elif schema["type"] == "email":
             self._validate_email(data_field, key)
-        elif schema["type"] == "instrumentlist":
-            self._validate_instrument_list(data_field)
+        elif schema["type"].endswith("list"):
+            self._validate_list(schema["type"], data_field)
```

Comment:
So you no longer validate that the instruments in the list make sense? Will try to come back to this comment later if I find the answer below.

Reply:
`self._validate_list(schema["type"], data_field)` is replacing ` self._validate_instrument_list(data_field)` by calling the more generic function; `schema["type"],` can be of type instrumentlist or documentlist. So there should be no change to before when the list type is `instrumentlist`

Reply:
Yeah I realised that later and figured it's a shame that we don't validate that the input is a valid instrument. It would open somewhat of a can of worms because the list of valid instruments might change with time and then we would have to change this validation function. However, perhaps it's something to consider. I will approve this PR anyway and let you decide if you want to add it. If not, simply merge.

Reply:
Wait - here we validate data types, not the actually values (this is consistent throughout the class). The name validation should happen somewhere, but not in the schema validation.